client/components/Header.js

```diff
@@ -1,18 +1,43 @@
 import React from "react";
-import { NavLink} from "react-router-dom"
+import { Link } from "react-router-dom"
 
 class Header extends React.Component {
+
+  handleLogout = () => {
+    localStorage.clear()
+    this.redirect()
+  }
+
+  redirect = () => {
```

Comment:
If `redirect` is being called only once then it shouldn't be an extra function just write it in `handleLogout` function itself.

Reply:
@AmanSharma4419 Can you please fix this?

Reply:
@akhildhiman Can you please fix this?

Reply:
@BadBrahmin I've updated it. Please check.